src/v2/Components/ArtworkFilter/SavedSearch/Components/CreateAlertButton.tsx

```diff
@@ -59,6 +59,7 @@ export const CreateAlertButton: React.FC<CreateAlertButtonProps> = ({
         },
         contextModule: ContextModule.artworkGrid,
         intent: Intent.createAlert,
+        redirectTo: location.href,
```

Comment:
we cannot apply `useRouter` here, because when the filters are changed, the `location` value is not updated. Alternatively, we can use the `buildUrl` helper. What do you think: leave the current implementation or is it better to use the `buildUrl` approach?

```javascript
const { filters } = useArtworkFilterContext()
...
const clearFilters = omit(filters, ["reset"])
const redirectTo = buildUrl(clearFilters)

openAuthToSatisfyIntent(mediator, {
  ...
  redirectTo,
})
```